sdk/textanalytics/Azure.AI.TextAnalytics.Protocol/demo/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using Azure;
+using Azure.AI.TextAnalytics.Protocol;
+using Azure.Core;
+using System;
+
+namespace DemoApp
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            TextAnalyticsClient client = new TextAnalyticsClient(new Uri("<fill-me-in>"), new AzureKeyCredential("<fill-me-in>"));
+
+            // Passing the body "in-line" and using the object constructor with an anonymous object.
+            {
+                DynamicResponse res = client.GetLanguages(new JsonData(new
+                {
+                    doucments = new JsonData(new object[]
```

Comment:
Also, what would it take to allow people to simply pass anonymous types to these LLC methods? Would we need to add an implicit cast from object to JsonData? What that work?

Reply:
> Also, what would it take to allow people to simply pass anonymous types to these LLC methods? Would we need to add an implicit cast from object to JsonData? What that work?

The C# compiler doesn't allow creating conversions to or from base-types (was news to me, I tried to add an implicit conversion to JsonData from object), so we can't have a conversion as nice as that would be.

We could also have an overload where we replace `JsonData` with `object` and it would just call `new JsonData()` on your behalf.

Reply:
If we added an overload taking dynamic (instead of object), would it allow passing in anonymous types?

Reply:
Yeah, you should be able to pass an anonymous object to a `dynamic` parameter.